wagtail/contrib/modeladmin/templatetags/modeladmin_tags.py

```diff
@@ -182,8 +182,3 @@ def result_row_value_display(context, index):
         'closing_tag': closing_tag,
     })
     return context
-
-
-@register.filter
-def get_content_type_for_obj(obj):
-    return obj.__class__._meta.verbose_name
```

Comment:
I think this filter may have been removed accidentally?

Reply:
Thanks, just put through some changes